### DISCUSSION TITLE
Theme looks good in titlebar

### DISCUSSION THREAD

**Discussion by littlewhitecloud** [rdbende/Sun-Valley-ttk-theme#68] [General]
![image](https://user-images.githubusercontent.com/71159641/209783374-f5ea3613-eb65-4ec3-9462-0eb7883d3bf9.png)
![image](https://user-images.githubusercontent.com/71159641/209128673-93a6f1db-66a6-4667-9bd7-a3c2ba096f5c.png)
With the blur, the window look really good!
![image](https://user-images.githubusercontent.com/71159641/210036942-a6d0423a-cf0e-41c4-8a6e-9c9ccb47e7e1.png)

**Comment by HuyHung1408** [comment]
looks good, I just checked out your repo, the searchbox in the titlebar is great ngl

**Reply by littlewhitecloud** [reply]
Thanks! I rebuild the titlebar and you can put anythings into it (except menu). This project is still in preview......

**Reply by littlewhitecloud** [reply]
By the way, can it run correctly on windows11? (I am using windows 10). Thanks for you reply!

**Reply by HuyHung1408** [reply]
I haven't tried it yet, I just moved to a new laptop which doesn't have Python installed, also I haven't touch Python for a while because exams.

**Reply by HuyHung1408** [reply]
I'll try if I'm free

**Reply by littlewhitecloud** [reply]
Ok, thanks~
